blog/_posts/2023-03-20-curation-model.md

```diff
@@ -0,0 +1,72 @@
+
+# The ADS Core Collection
+For the Core Collection we add the filter that these journals are the top N of the ranked list, such that these journals represent at least 95% of the total sum of Eigenfactor values. This is a meaningful filter, since Eigenfactor values are additive.
+
+The next question is whether it will be sufficient to just accept these lists "as is"? Obviously, these lists need curation. The ADS curation team needs to assess the lists generated. The most likely scenario is that the team feels that a particular journal belongs in the Core Collection, even though the algorithm did not select it. For example, it is possible that a journal with a lot of cross-disciplinary content did not get included. While the ADS citation network is the best one available for core astronomy disciplines, it has known gaps (sometimes significant) for disciplines in the "Outer Rim" of astronomy and astrophysics (like astrochemistry, astrobiology, astrophotonics, archaeoastronomy and astroinformatics). It is not uncommon for articles in a journal like Astrobiology to have reference lists in the ADS that are only 50% complete. The ADS is also aware that IEEE content is very important for many of its users; by identifying IEEE content as belonging to the Core Collection we can assure completeness will become a top priority. Another possibility is that a journal is too "young" to have acquired a position in the citation network to get a sufficiently large Eigenfactor value; in this case it is purely a qualitative argument to still include it in the Core Collection, assuming that eventually it will position itself as an influential journal within the citation network. 
+
+The algorithm, in combination with additional curation work, has resulted in a list of 207 journals.
+
+We will reevaluate this list on a regular basis. The stage of research fields and publication venues is a dynamic one; boundaries between disciplines become less distinct, multimessenger science becomes more prolific and techniques from one discipline turn out to be highly applicable to other ones (like e.g. [one developed in astrophysics to cancer research](https://ui.adsabs.harvard.edu/abs/2021IOIns...2..247T/abstract)).
```

Comment:
How frequently will you reevaluate this list? Will journals that are currently in the core collection ever move out of it, or is the anticipation that you'll only add journals to the core collection and not subtract?